code/datums/mutantraces.dm

```diff
@@ -761,7 +763,7 @@
 
 			if (C && C.points)
 				if (last_drain + 30 <= world.time)
-					C.points = max(0, C.points - 1)
+					C.points = max(0, C.points - 1*mult)
```

Comment:
not spacing your operators :shelterfrog:

Reply:
also, ambiguous evaluation due to lack of parens

Reply:
_fine_